src/components/Login/helpers/TokenService.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable require-jsdoc */
+interface IUserToken {
+  accessToken: string;
+  refreshToken: string;
+}
+
+interface IUserInfo {
+  userId: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  isActivated: boolean;
+  tokens: IUserToken;
+}
+class UserTokenService {
+  getUserInfo(): IUserInfo | undefined {
+    if (typeof window !== "undefined") {
+      const userInfo = JSON.parse(window.localStorage.getItem("UserInfo") as string);
```

Comment:
Add "try catch" to wrap "JSON.parse" to handle getItem Error

Reply:
Added try catch for handle getItem error.